docs/api-integration/tlm_event_socket.md

```diff
@@ -0,0 +1,308 @@
+# Tracim Live Message
+
+## Definition of a TLM
+
+The real-time architecture of tracim relies on a socket which dispatch all events generated by users actions.  
+These events are dispatched on a per-user basis and are so called TLM - Tracim Live Messages.
+
+You can connect to socket to listen to tracim activity and trigger actions on any piece of software.
+
+A few use case examples:
+
+- push a file to a web server when a given user comments "VALID FOR PROD" on the content,
+- trigger an email to security staff when someone is invited on a tracim workspace
```

Comment:
Here, line not finish with `.` but with `,`. On some other doc, this same kind of line are finished by `.`  or wiht nothing.

Reply:
fixed